imagetest/test_suites/network/setup.go

```diff
@@ -74,6 +78,16 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 	}
 	vm2.UseGVNIC()
 	vm1.RunTests("TestPingVMToVM|TestDHCP|TestDefaultMTU")
-	vm2.RunTests("TestAlias|TestGVNIC")
+	vm2.RunTests("TestAlias")
+	if strings.Contains(t.Image, "debian-10") {
```

Comment:
this is interesting. how should we maintain this going forward? one future improvement might be to  read the features from the image at this point

Reply:
I tried to read from the Image the GuestOsFeatures but got a null error https://pkg.go.dev/google.golang.org/api/compute/v1#GuestOsFeature

Reply:
discussed offline, will follow up in another PR